types/config.d.ts

```diff
@@ -0,0 +1,38 @@
+interface dataFormat{
+    keyType : string,
+    valueType : string
+}
+//TODO : Add option for key type and mediatype among the three
+
+interface authentication{
+    enabled: boolean,
+    saslMechanism: string,
+    userName: string,
+    password: string,
+    serverName: string,
+}
+
+export interface entry{
+    key: mediaType,
+    value: mediaType
+}
+
+export interface statsItem{
+    STAT_NAME : string,
+    STAT_VALUE : string
+}
+
+export type mediaType = string | number | object ;
+//TODO: add type for protobuf instance
+
+export interface addr{
+    port : Number,
+    host : String
```

Comment:
do we need capital letters here?

Reply:
No, we don't. It was added by mistake. Actually, it's a good practice to use lowercase only. I will change it to lowercase.

Reply:
Hi @NeerajGartia21 @rigazilla 

yes, also related to capital letters if you plan to continue manually creating the `.d.ts` files, you would also need to keep the first letter of the interfaces in uppercase.